javaslang/src/main/java/javaslang/collection/LinkedHashMap.java

```diff
@@ -92,28 +92,198 @@ private LinkedHashMap(Queue<Tuple2<K, V>> list, HashMap<K, V> map) {
+     * @param e6  The 6th entry
+     * @return A new Map containing the given entries
+     */
+    @SuppressWarnings("unchecked")
+    public static <K, V> LinkedHashMap<K, V> ofEntries(Tuple2<? extends K, ? extends V> e1, Tuple2<? extends K, ? extends V> e2, Tuple2<? extends K, ? extends V> e3, Tuple2<? extends K, ? extends V> e4, Tuple2<? extends K, ? extends V> e5, Tuple2<? extends K, ? extends V> e6) {
+        final HashMap<K, V> map = HashMap.ofEntries(e1, e2, e3, e4, e5, e6);
+        final Queue<Tuple2<K, V>> list = Queue.of((Tuple2<K, V>) e1, (Tuple2<K, V>) e2, (Tuple2<K, V>) e3, (Tuple2<K, V>) e4, (Tuple2<K, V>) e5, (Tuple2<K, V>) e6);
+        return new LinkedHashMap<>(list, map);
+    }
+
+    /**
+     * Creates a {@code LinkedHashMap} of the given entries.
+     *
+     * @param <K> The key type
+     * @param <V> The value type
+     * @param e1  The 1st entry
+     * @param e2  The 2nd entry
+     * @param e3  The 3rd entry
+     * @param e4  The 4th entry
+     * @param e5  The 5th entry
+     * @param e6  The 6th entry
+     * @param e7  The 7th entry
+     * @return A new Map containing the given entries
+     */
+    @SuppressWarnings("unchecked")
+    public static <K, V> LinkedHashMap<K, V> ofEntries(Tuple2<? extends K, ? extends V> e1, Tuple2<? extends K, ? extends V> e2, Tuple2<? extends K, ? extends V> e3, Tuple2<? extends K, ? extends V> e4, Tuple2<? extends K, ? extends V> e5, Tuple2<? extends K, ? extends V> e6, Tuple2<? extends K, ? extends V> e7) {
+        final HashMap<K, V> map = HashMap.ofEntries(e1, e2, e3, e4, e5, e6, e7);
+        final Queue<Tuple2<K, V>> list = Queue.of((Tuple2<K, V>) e1, (Tuple2<K, V>) e2, (Tuple2<K, V>) e3, (Tuple2<K, V>) e4, (Tuple2<K, V>) e5, (Tuple2<K, V>) e6, (Tuple2<K, V>) e7);
+        return new LinkedHashMap<>(list, map);
+    }
+
+    /**
+     * Creates a {@code LinkedHashMap} of the given entries.
+     *
+     * @param <K> The key type
+     * @param <V> The value type
+     * @param e1  The 1st entry
+     * @param e2  The 2nd entry
+     * @param e3  The 3rd entry
+     * @param e4  The 4th entry
+     * @param e5  The 5th entry
+     * @param e6  The 6th entry
+     * @param e7  The 7th entry
+     * @param e8  The 8th entry
+     * @return A new Map containing the given entries
+     */
+    @SuppressWarnings("unchecked")
+    public static <K, V> LinkedHashMap<K, V> ofEntries(Tuple2<? extends K, ? extends V> e1, Tuple2<? extends K, ? extends V> e2, Tuple2<? extends K, ? extends V> e3, Tuple2<? extends K, ? extends V> e4, Tuple2<? extends K, ? extends V> e5, Tuple2<? extends K, ? extends V> e6, Tuple2<? extends K, ? extends V> e7, Tuple2<? extends K, ? extends V> e8) {
+        final HashMap<K, V> map = HashMap.ofEntries(e1, e2, e3, e4, e5, e6, e7, e8);
+        final Queue<Tuple2<K, V>> list = Queue.of((Tuple2<K, V>) e1, (Tuple2<K, V>) e2, (Tuple2<K, V>) e3, (Tuple2<K, V>) e4, (Tuple2<K, V>) e5, (Tuple2<K, V>) e6, (Tuple2<K, V>) e7, (Tuple2<K, V>) e8);
```

Comment:
in case you will keep these, consider creating a method `t` here also, just for the casts :)